examples/deadline/Local-Zone/ts/bin/app.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import 'source-map-support/register';
+import {
+  MachineImage,
+} from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+
+import { config } from './config';
+import { ComputeTier } from '../lib/compute-tier';
+import { NetworkTier } from '../lib/network-tier';
+import { SecurityTier } from '../lib/security-tier';
+import { ServiceTier } from '../lib/service-tier';
+
+  // ------------------------------ //
+  // --- Validate Config Values --- //
+  // ------------------------------ //
+  if (!config.keyPairName) {
+    console.log('EC2 key pair name not specified. You will not have SSH access to the render farm.');
+  }
+
+  if (config.deadlineClientLinuxAmiMap === {['region']: 'ami-id'}) {
```

Comment:
We can remove this since it was a safeguard for when the `config.ts` file didn't provide default AMI IDs.

Reply:
updated